astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx

```diff
@@ -9,38 +9,38 @@ tags: webauthn, passkeys, passwordless
 excerpt_separator: "{/* more */}"
 ---
 
-Unless you use two-factor authentication (2FA) with your password logins, you're prone to cyberattacks. This is where [Web Authentication (WebAuthn)](https://webauthn.guide) can help.
+Unless you use two-factor authentication (2FA) with your password logins, you're prone to cyberattacks. This is where [Passkeys (WebAuthn)](https://webauthn.guide) can help.
 
-WebAuthn is an authentication standard that uses [asymmetric cryptographic](https://www.techtarget.com/searchsecurity/definition/asymmetric-cryptography) keys to authenticate users instead of passwords, mitigating cyberattacks. With WebAuthn, users can authenticate using their devices (with biometrics) without having to remember their passwords, store them, or worry about them getting compromised. The WebAuthn credentials are also known as "passkeys".
+WebAuthn is an authentication standard that uses [asymmetric cryptographic](https://www.techtarget.com/searchsecurity/definition/asymmetric-cryptography) keys to authenticate users instead of passwords, mitigating cyberattacks. With WebAuthn, users can authenticate using their devices (with passkeys) without having to remember their passwords, store them, or worry about them getting compromised.
 
 {/* more */}
 
-In this guide, you'll learn more about WebAuthn and why it matters. You'll also explore some use cases where WebAuthn is important and see how it can help prevent cyberattacks.
+In this guide, you'll learn more about passkeys and why they matter. You'll also explore some use cases where passkeys are important and see how it can help prevent cyberattacks.
 
-## What Is WebAuthn
+## What Are Passkeys?
 
 User’s passwords are often weak, reused, and can be easily hijacked. The site ["have i been pwned?"](https://haveibeenpwned.com) has over 11 billion accounts that have been compromised. Because of this, they're not suited for guarding sensitive accounts like banking or social media applications. While MFA offers additional security, WebAuthn is even better when it comes to guarding sensitive data since it uses passwordless authentication.
 
-WebAuthn provides a means for servers to authenticate users using cryptographic keys (private and public) in place of passwords. The private and public keys are used to authenticate users, and the public key is stored in the server (also known as a relying party or RP), while the private key is stored locally in the user's device.
+Passkeys provide a means for servers to authenticate users using cryptographic keys (private and public) in place of passwords. The private and public keys are used to authenticate users, and the public key is stored in the server (also known as a relying party or RP), while the private key is stored locally in the user's device.
```

Comment:
This swap doesn't work. 

Passkeys are credentials that allow a user to prove their identity, while WebAuthn is a standard javascript API that allows an application to create and use passkeys for sign-in. This is specifically talking about WebAuthn and not passkeys.

Reply:
Reverted